grunttasks/l10n-generate-pages.js

```diff
@@ -111,19 +117,21 @@ module.exports = function (grunt) {
         var terms = legalTemplates[context.lang].terms || legalTemplates[defaultLegalLang].terms;
         var privacy = legalTemplates[context.lang].privacy || legalTemplates[defaultLegalLang].privacy;
         var template = Handlebars.compile(contents);
-        var out = template({
+        var data = {
           fontSupportDisabled: context.fontSupportDisabled,
           l10n: context,
           lang: context.lang,
           lang_dir: context.lang_dir, //eslint-disable-line camelcase
           locale: context.locale,
-          // Re-insert the message tag to allow the node server
-          // to render the error message at render time.
-          message: '{{ message }}',
           privacy: privacy,
           terms: terms
+        };
+        // Propagate any tags that are required for data
+        // to be rendered dynamically by the server.
+        PROPAGATED_TEMPLATE_FIELDS.forEach(function (field) {
```

Comment:
I was thinking of doing the exact same thing with message and thought [YAGNI](https://blog.codinghorror.com/kiss-and-yagni/). In the future we could have both `safe` and `unsafe` variants, `safe` using `{{field}}`, unsafe using `{{{field}}}`